apps/sms/test/marionette/lib/messages_activity_caller.js

```diff
@@ -60,15 +60,16 @@
           client.helper.waitForElement(SELECTORS.pickContactButton).tap();
         },
 
-        sendMessage: function(number, body) {
+        sendMessage: function(parameters) {
           var button = client.helper.waitForElement(
             SELECTORS.sendMessageButton
           );
 
-          button.scriptWith(function(element, number, body) {
-            element.dataset.number = number;
-            element.dataset.body = body;
-          }, [number, body]);
+          button.scriptWith(function(element, parameters) {
+            Object.keys(parameters).forEach(function(parameterKey) {
+              element.dataset[parameterKey] = parameters[parameterKey];
+            });
```

Comment:
BTW I wonder if it would have been more readable to call a method directly instead of tapping a button.